packages/data-stores/src/wpcom-plans-ui/actions.ts

```diff
@@ -16,7 +20,7 @@ export const setSelectedStorageOptionForPlan = ( {
 	planSlug,
 	siteId,
 }: {
-	addOnSlug: WPComStorageAddOnSlug;
+	addOnSlug: WPComStorageAddOnSlug | WPComPlanStorageFeatureSlug;
```

Comment:
I think `addOnSlug` should now be renamed now that the real type is reflected. It can either be the add-on slug or a feature slug. We didn't make this distinction previously and worked the code around it from the looks of it. This is now the "selected storage option slug"

cc @jeyip if you can confirm :-)

Reply:
I'll take a look later today 🙂

Reply:
I'm not sure if things changed between now and my original implementation, but I had originally set up the selected storage option slice of state to only be meaningful if a 50GB or 100GB storage add-on was selected. Otherwise, I had used some false-y value to represent that no add-on was chosen and that the plan's default storage option was currently selected.

Reply:
Thanks @jeyip . It makes sense. The implementation evolved properly, meaning we do populate this variable with either the add-on/upgrade feature or the standard plan feature. The current changes aren't touching the implementation side. Just adjusting the tuypes a little. 

It might just be a case of renaming to `slug`. I'll revisit this in a follow-up.